.github/workflows/releases.yml

```diff
@@ -25,29 +27,38 @@ jobs:
         env:
           GH_OAUTH_CLIENT_ID: 178c6fc778ccc68e1d6a
           GH_OAUTH_CLIENT_SECRET: ${{secrets.OAUTH_CLIENT_SECRET}}
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
```

Comment:
goreleaser now needs to write to `github/homebrew-gh` directly, so regular GITHUB_TOKEN won't suffice as it only has privileges to write to this repo